build.gradle

```diff
@@ -418,6 +418,7 @@ project(':iceberg-aws') {
     testClassesDirs = sourceSets.integration.output.classesDirs
     classpath = sourceSets.integration.runtimeClasspath
   }
+  check.dependsOn integrationTest
```

Comment:
I think it needs AWS credentials, just enabling will not work.

have you seen this?
https://github.com/apache/iceberg/pull/4855